dags/bqetl_pocket.py

```diff
@@ -49,8 +49,9 @@
         project_id="moz-fx-data-shared-prod",
         owner="jklukas@mozilla.com",
         email=["jklukas@mozilla.com", "telemetry-alerts@mozilla.com"],
-        date_partition_parameter="submission_date",
+        date_partition_parameter=None,
         depends_on_past=False,
+        parameters=["submission_date:DATE:{{ds}}"],
```

Comment:
This is our pattern for a `query.sql` on a non-partitioned table. The job is currently failing because it's using the `date_partition_parameter` to try to decorate the destination table with a date partition.